subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -0,0 +1,73 @@
+project('cmark-gfm', 'c', version: '0.29.0.gfm.6', default_options: ['warning_level=3'])
+
+cdata = configuration_data({
+  'HAVE_STDBOOL_H': true,
+  'HAVE___BUILTIN_EXPECT': true,
+  'HAVE___ATTRIBUTE__': true,
+})
+
+configure_file(
+  format: 'cmake@',
+  input: 'src/config.h.in',
+  output: 'config.h',
+  configuration: cdata)
+
+varr = meson.project_version().split('.')
+vdata = configuration_data({
+  'PROJECT_VERSION_MAJOR' : varr[0],
+  'PROJECT_VERSION_MINOR' : varr[1],
+  'PROJECT_VERSION_PATCH' : varr[2],
+  'PROJECT_VERSION_GFM' : varr[4],
+})
+
+configure_file(
+  format: 'cmake@',
+  input: 'src/cmark-gfm_version.h.in',
+  output: 'cmark-gfm_version.h',
+  configuration: vdata)
+
+src = [
+  'src/cmark.c',
```

Comment:
Should use files() here. Allows muon's FMT to alphabetize the entries.

Reply:
Fixed, thank you.